apps/prairielearn/assets/scripts/instructorAiGenerateDraftsSampleQuestions.tsx

```
@@ -0,0 +1,83 @@
+                                        style="white-space: normal;"
+                                    >
+                                        {examplePrompts[0].name}
+                                    </span>
+                                </button>
+                                <div class="dropdown-menu py-0">
+                                    <div class="overflow-auto">
+                                        {examplePrompts.map((a, index) => {
+                                        return <a
+                                                class="dropdown-item"
+                                                data-id={a.id}
+                                            >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)